docs/docs/references/connectors/README.mdx

```diff
@@ -0,0 +1,106 @@
+import ConnectorPlatformIntro from './fragments/_connector_platform_intro.mdx';
+import ConnectorStorageIntro from './fragments/_connector_storage_intro.mdx';
+import ConnectorTypeIntro from './fragments/_connector_type_intro.mdx';
+
+# 🔌 Connectors
+
+## Introduction
+
+### What is a connector?
+
+_Connectors_ play a critical role in Logto. With their help, Logto enables end-users to use SMS-based or email-based passwordless registration and the capabilities of signing in with social accounts. With the increasing popularity of websites and applications, passwordless and social sign-ins allow users to avoid managing numerous accounts and passwords.
```

Comment:
`enables end-users to use SMS-based or email-based ...` sounds not appropriate.

Reply:
deleted to make it more clear.